src/utils/oidcConfig.js

```diff
@@ -0,0 +1,36 @@
+/* eslint-disable sonarjs/no-nested-template-literals */
+import config from "../config";
+
+const port = window.location?.port || '';
+const baseUrl = `${window.location.protocol}//${window.location.hostname}${port ? `:${  port}` : ''}`;
+
+const commonOidcConfig = {
+  silent_redirect_uri: `${baseUrl}/silent-renew/`,
+  authority: config.openIdAuthority,
+  redirect_uri: `${baseUrl}/callback`,
+  post_logout_redirect_uri: `${baseUrl}/callback/logout`,
+}
+
+// profiili config
+export const userOidcConfig = {
+  client_id: config.openIdClientId,
+  scope: 'openid profile email',
+  ...commonOidcConfig,
+}
+
+export const adminOidcConfig = {
+  client_id: 'kerrokantasi-ui-staff-dev',
+  scope: 'openid profile',
+  ...commonOidcConfig,
+}
+
+export const apiTokenClientConfig = {
+  url: 'https://tunnistus.test.hel.ninja/auth/realms/helsinki-tunnistus/protocol/openid-connect/token',
+  queryProps: {
+    grantType: 'urn:ietf:params:oauth:grant-type:uma-ticket',
+    permission: '#access',
+  },
+  audiences: ['kerrokantasi-api-dev'],
```

Comment:
Audiences should probably also come from config.

Reply:
fixed.